tests/test_stt.py

```diff
@@ -9,7 +9,7 @@
 
 import pytest
 from livekit import agents, rtc
-from livekit.plugins import azure, deepgram, google, openai, silero
+from livekit.plugins import assemblyai, azure, deepgram, google, openai, silero
```

Comment:
since we add this new dependency, we need to add `./livekit-plugins/livekit-plugins-assemblyai \` to the `tests.yml` file

Reply:
Nice work so far, care to join a shared slack channel to help get this over the finish line? Just need your emails